src/ol/style/style.js

```diff
@@ -19,6 +19,7 @@ goog.require('ol.style.Stroke');
  * feature or layer that uses the style is re-rendered.
  *
  * @constructor
+ * @struct
```

Comment:
To prevent the use of properties not declared in the constructor. We use on every class except here and in ol.format.Format